### PR TITLE
minor/feature/UBI-514-traces-and-metrics

### PR DESCRIPTION
**Pull request by ArcticWalrus**
#### What changes does this PR contain?
Added telemetry exports
#### Which JIRA ticket ID's are associated with this PR?
UBI-514

### CODE REVIEW

**Review by JeremiSavard — APPROVED** [review]

**Comment by ArcticWalrus** [comment]
@JeremiSavard @MaximeStOnge Don't merge this if possible, I'm going to use this PR as a template to compare the others to. I won't assign you on the other ones until they are all ready

**Pull request merged**
